addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -103,475 +126,456 @@
-                            <i class="fa fa-circle-o fa-stack-2x"/>
-                            <i class="fa fa-check fa-stack-1x"/>
-                        </a>
+    <div t-attf-class="o_wforum_#{post_type} row g-0 mb-2 rounded #{can_bump and 'oe_js_bump'}
+                       #{ 'o_wforum_answer_correct my-2 mx-n3 mx-lg-n2 mx-xl-n3 py-3 px-3 px-lg-2 px-xl-3' if post_id == answer and post_id.is_correct else '' }"
+         t-att-data-type="post_type"
+         t-att-data-last-update="post_id.write_date"
+         t-attf-data-id="#{post_type}-#{post_id.id}"
+         t-attf-id="#{post_type}-#{post_id.id}"
+         t-att-data-state="post_id.state">
+            <div class="d-flex flex-column col-auto pe-2 pe-lg-3">
+                <t t-call="website_forum.author_box">
+                    <t t-set="object" t-value="post_id"/>
+                    <t t-set="show_image" t-value="True"/>
+                </t>
+                <div t-attf-class="align-self-center flex-grow-1 mt-2 border-start opacity-50 #{post_id.is_correct and 'border-success'}"/>
+            </div>
+            <div class="post_content_wrapper col">
+                <header class="o_wforum_post_header d-flex align-items-center mb-1">
+                    <t t-call="website_forum.author_box">
+                        <t t-set="object" t-value="post_id"/>
+                        <t t-set="allow_biography" t-value="True"/>
+                        <t t-set="show_name" t-value="True"/>
                     </t>
-                </div>
-                <div class="col">
-                    <div class="o_wforum_answer_header d-flex align-items-start mb-2">
-                        <t t-call="website_forum.author_box">
-                            <t t-set="object" t-value="answer"/>
-                            <t t-set="show_name" t-value="True"/>
-                            <t t-set="show_date" t-value="True"/>
-                            <t t-set="allow_biography" t-value="True"/>
-                            <t t-set="object_validable" t-value="True"/>
-                        </t>
-                        <span class="o_wforum_answer_correct_badge border small border-success rounded-pill fw-bold text-success ms-2 px-2">
+                    <!-- [TODO] To be replaced with relative date/time (ex: "2 days ago") -->
+                    <span class="ms-2 opacity-75 small text-muted">
+                        <!-- <time  t-field="post_id.create_date" t-options='{"format": "d MMMM y "}'/>
+                        at
+                        <time t-field="post_id.create_date" t-options='{"format": "HH:mm "}'/> -->
+                        2d ago
+                    </span>
+                    <t t-if="answer">
+                        <span t-if="_question_creator == _answer_creator" class="badge ms-2 border border-dark rounded-pill px-2 py-1 text-reset">
+                            Author
```

Comment:
Did not try the UI for this one, we have a "Author" without anything else displayed ?